Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -151,7 +151,6 @@ cme,GNF,future,Nonfat Dry Milk Futures,USD,44000.0,0.00025,1.0,,1,100
 cme,HE,future,Lean Hog Futures,USD,40000.0,0.00025,1.0,,1,100
 cme,IBV,future,USD-Denominated Ibovespa Index Futures,USD,1.0,5.0,1.0
 cme,J7,future,E-mini Japanese Yen Futures,USD,6250000.0,1e-06,1.0
-cme,LBR,future,Lumber Futures,USD,27.5,0.50,1.0
```

Comment:
Whops seems this was generated with an outdated SPDB, this shouldn't be removed?